gateway/1.6/modules/ROOT/pages/policies-reorder.adoc

```diff
@@ -1,18 +1,60 @@
 
-To execute certain policies before others, you can configure the order of execution of included and custom API-level policies. However, you cannot configure API-level policies to execute before automated policies.
+To execute certain policies before others, you can configure the order of execution of included and custom policies. 
 
-For Flex Gateway running in Connected Mode, you reorder policies via API Manager. For Flex Gateway running in Local Mode you order policies via a `PolicyBinding` YAML resource.
+You can configure both included and custom policies to execute as automated policies or API-level policies. Automated policies are policies that are applied to all API instances deployed on the Flex Replica and execute before API-level policies. API-level policies are policies that are only applied to a specific API instance.
+
+For Flex Gateway running in Connected Mode, you order policies via API Manager. For Flex Gateway running in Local Mode you order policies via a `PolicyBinding` YAML resource.
 
 IMPORTANT: You cannot configure the execution order of the xref:policies-included-cors.adoc[Cross-Origin Resource Sharing (CORS) policy]. The CORS policy executes before automated and API-level policies.
 
-== Flex Gateway in Connected Mode
+== Policy Execution Order
+
+Policies can either execute on the API request, the API response, or on both. Policies execute in order during the API request and in inverse order during the API response. If a policy does not execute in a certain direction, Flex Gateway skips the policy in that direction.
+
+For example, if you have four policies ordered:
+
+. Automated Policy 1
+. Automated Policy 2
+. API-Level Policy 1
+. API-Level Policy 2
+
+The policies execute in the following order on the response:
+
+. API-Level Policy 2
+. API-Level Policy 1
+. Automated Policy 2
+. Automated Policy 1
+
+[[automated-policies]]
+== Order Automated Policies
+
+Refer to the following sections for your relevant mode.
+
+=== Flex Gateway in Connected Mode
+
+You cannot configure the order of automated policies in API Manager. Automated policies execute in the order they are applied to the API instance. Apply automated policies in the order you want them to execute.
+
+=== Flex Gateway in Local Mode
+
+To apply a automated policy to Flex Gateway running in Local Mode, you must apply a resource-level policy. By doing so, you can apply the same policy to all API instances running on the Flex Replica.
+
+To apply a resource-level policy to all API instances, refer to xref:flex-local-secure-api-with-auto-policy.adoc[].
+
+To order automated resource-level policies, edit the `order` parameter to match when you want the policy to execute. Ensure that this order does not conflict with the order of any other policies. For example, if you apply two automated resource-level policies with `order: 1` and `order: 2`, any API-level policy must start at `order: 3`.
```

Comment:
I keep reading "To order" and wanting it to say "To specify the order of", but I'm not sure why. I think because the verb order means something else to me.